anchor/network/src/network.rs

```diff
@@ -113,6 +115,7 @@ impl<R: MessageReceiver> Network<R> {
         outcome_rx: mpsc::Receiver<Outcome>,
         executor: TaskExecutor,
         spec: Arc<ChainSpec>,
+        dynamic_target_peers: bool,
```

Comment:
```suggestion
        target_peers: bool,
```
they can also be static right? If they are user provided